Containerfile

```diff
@@ -43,11 +43,21 @@ if [ "${OPENSHIFT_CI}" != 0 ]; then
   /run/src/ci/get-ocp-repo.sh /etc/yum.repos.d/ocp.repo
 fi
 
+source /etc/os-release
+
+# XXX: For SCOS, only allow certain packages to come from ART; everything else
+# should come from CentOS. We should eventually sever this.
+if [ $ID = centos ]; then
+  awk "/\[.*\]/{p=0} /\[rhel-9.6-server-ose-4.19\]/{p=1} p" /etc/yum.repos.d/*.repo > /etc/yum.repos.d/okd.repo.tmp
+  sed -i -e 's,rhel-9.6-server-ose-4.19,rhel-9.6-server-ose-4.19-okd,' /etc/yum.repos.d/okd.repo.tmp
+  echo 'includepkgs=openshift-* ose-aws-ecr-* ose-azure-acr-* ose-gcp-gcr-*' >> /etc/yum.repos.d/okd.repo.tmp
```

Comment:
does space separated work here? The ones in `rhel-9.6.repo` are comma separated.


also, I thought `>>` didn't work in buildah?

Reply:
> does space separated work here? The ones in rhel-9.6.repo are comma separated.

Yeah, both work it seems. But I'll switch it over to comma too for consistency.